sdks/nodejs/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@google-cloud/agones-sdk",
-  "version": "1.15.0-dev",
+  "version": "1.16.0-dev",
```

Comment:
Not sure why this increment is here? Removing the change would help with conflicts.

Reply:
I don't know why it's changed. Seems some automatic steps in the build process. But I change it back now.